server/command/unsubscribe.go

```diff
@@ -3,7 +3,7 @@
 
 package command
 
-func (c *Command) unsubscribe(parameters ...string) (string, bool, error) {
+func (c *Command) unsubscribe(_ ...string) (string, bool, error) {
```

Comment:
Same here and at the other methods.

Reply:
To be clear, you're suggesting we change the commands to remove the unused parameter. Should we do the same for the `settingspanel`? I'm thinking we should keep the `settingspanel` as-is, to avoid having to change any logic calling the "steps" of the panel